.github/workflows/ci_unit_tests.yaml

```diff
@@ -58,7 +63,7 @@ jobs:
         sudo chmod -R a+w /scratch1/NCEPDEV
         source rocoto-${{ env.ROCOTO_VERSION }}/rocoto_path.sh
         cd global-workflow/sorc
-        git submodule update --init -j 2 wxflow ufs_model.fd
+        git submodule update --init -j 2 ufs_model.fd gsi_monitor.fd
```

Comment:
Why do we need gsi_monitor in the CI Unit Tests?

Reply:
When a CI Unit test fails, it shows that these files from gsi_monitor are sourced in configs and are missing. E.g. [here](https://github.com/aerorahul/global-workflow/actions/runs/15879215732/job/44774707069#step:8:299)